bot/cogs/wolfram.py

```diff
@@ -1,21 +1,18 @@
-import aiohttp
 import discord
-import os
 from io import BytesIO
 from urllib import parse
 from discord.ext import commands
 from bot.settings import APPID
 
 
 class Wolfram(commands.Cog):
-    """command for wolfram search"""
+    """Command for Wolfram search."""
     def __init__(self, bot):
         self.bot = bot
         self.query = "http://api.wolframalpha.com/v2/{request}?{data}"
         
 
-    # wolfram command, takes in a search and gives the result
-    # PS, I didn't totally steal it from seasonal bot
+    """Wolfram command, takes in a search and gives the result."""
     @commands.command(brief="Takes in a wolfram search and displays the result", usage=".wolfram [query]")
     async def wolfram(self, ctx, *, query):
 
```

Comment:
The docstring you added on line 15 should go inside the function, on the first line.

I should have clarified this earlier, sorry.

Reply:
alrighty np